### PR TITLE
ElasticMemCache

### PR DESCRIPTION
**Pull request by dieend**

### CODE REVIEW

**Comment by petrabarus** [comment]
I prefer something like

```
     [
      'class' => '\Urbanindo\Yii\Component\Cache\ElasticMemCache',
      'keyPrefix' => 'prefix',
      'hashKey' => false,
      'configServer' => [
           'host' => 'urbanindo.g4eo3j.cfg.apse1.cache.amazonaws.com',
           'port' => 11211,
      ]

```

and then in the class

```
    public function init(){
         $this->setServers($this->getNodes());
    }

   private function getNodes(){
       //load node names using the server configs and parse
   }
```

File: ElasticMemCache.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ *
+ * @author Adinata <mail.dieend@gmail.com>
+ * @since 2014.12.09
+ */
+class ElasticMemCache extends \CMemCache
+{
+    private $_cache = null;
+    public $cacheTime = 60;
+
+    public function setCache($config) {
```
Comment (petrabarus) [inline ElasticMemCache.php:26]:
```

public $configServer;

public function init(){
   $this->setServers($this->loadNodes());
   parent::init();
}

```

**Comment by dieend** [comment]
@petrabarus  Updated. please recheck.

**Comment by dieend** [comment]
@petrabarus fixed removing setter override.

**Pull request merged**